app/views/layouts/application.html.erb

```diff
@@ -10,9 +10,11 @@
     <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
   </head>
 
-  <body><%= render "shared/navbar" %>
-<%= render "shared/flashes" %>
-
+  <body>
+    <%= render "shared/navbar" %>
+    <%= render "shared/flashes" %>
+  </div>
```

Comment:
Maybe this `</div>` is not needed

Reply:
ah, I got it. was copying the code from Airbnb, and there was a container-all in Grabnb's code. 
Do you think it will be a good idea to contain all the views in a container div? 

```
`<body>
    <%= render "shared/navbar" %>
    <%= render "shared/flashes" %>
    <% if user_signed_in? %>
      <h2> Hi <%= current_user.email %></h2>
    <% end %>
    <div class="container all">
      <%= yield %>
    </div>
    <%= render "shared/footer" %>
  </body>`
```

Reply:
If you see that it looks better on your end, then ok to add. Otherwise, I think we can leave out the `container` from here.

Reply:
will remove the additional `container` for now.